.gitignore

```diff
@@ -64,6 +64,7 @@ nosetests.xml
 coverage.xml
 *.cover
 .hypothesis/
+src/dgtest/
```

Comment:
Why is this necessary? Are we cloning `dgtest` still?

Reply:
I was running it locally in a container and it spit out this artifact.

Reply:
huh is that to be expected? I'm cool with it but just not what I expected.

Reply:
I can't say for sure. But this was within a clean docker container.